### PR TITLE
[studio-ui] Template Code Examples #3430

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/3430

### CODE REVIEW

**Review by rart — APPROVED** [review]

File: src/main/webapp/repo-bootstrap/global/configuration/samples/sample-code-editor-config.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU General Public License
+  ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+
+<code-editor-config>
+  <theme>light</theme>       <!-- default theme: ligth | dark -->
```
Comment (sumerjabri) [inline src/main/webapp/repo-bootstrap/global/configuration/samples/sample-code-editor-config.xml]: Is the `theme` overridable? If so, why isn't it spec'd out in the overrides?

**Review by sumerjabri — CHANGES_REQUESTED** [review]
Include `<theme>` in all blueprint examples.

**Comment by jvega190** [comment]
@sumerjabri Done. Theme added

**Review by rart — CHANGES_REQUESTED** [review]
Please reformat the commented snippets as open comment, line break, snippet, line break, close comment.

e.g.
```xml
<!--
<snippet>
  ...
</snippet>
-->
```

**Review by rart — APPROVED** [review]

File: src/main/webapp/repo-bootstrap/global/blueprints/2000_headless_store/config/studio/code-editor-config.xml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU General Public License
+  ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+
+<code-editor-config>
+  <theme>light</theme>       <!-- default theme: ligth | dark -->
```
Comment (sumerjabri) [inline src/main/webapp/repo-bootstrap/global/blueprints/2000_headless_store/config/studio/code-editor-config.xml]: `light` is misspelled in the comments.

Also, why is that not commented out and use the default inherited setting?

**Review by sumerjabri — CHANGES_REQUESTED** [review]

**Review by sumerjabri — APPROVED** [review]

**Pull request merged**